scale/product/configuration/product_data_file.py

```diff
@@ -74,11 +76,17 @@ def _calculate_remote_path(self, job_exe, input_file_ids):
         :type input_file_ids: set of int
         :returns: The remote path for storing the products
         :rtype: str
-        '''
-
+        """
+
+        remote_path = ''
+        recipe = Recipe.objects.get_recipe_for_job(job_exe.job_id)
+        if recipe:
```

Comment:
It's not handled in a way. A recipe and job with the same type name will put products in the same top level directory (job type name/recipe type name). This doesn't really break anything, but it could be confusing if someone was browsing the file system. In practice I don't think it would occur very often, if at all. We could do some type of a prefix to distinguish recipes and jobs, but I'm not sure John and Chris would like that. Bigger picture I don't think that "browsing" the file system should be a normal use case. I can ask John and Chris about the prefix directories.